src/common.cpp

```diff
@@ -2065,7 +2066,8 @@ __declspec(dllexport) void *new_dll_graphics_driver(void *game_object)
 	info("Original resolution %ix%i, New resolution %ix%i\n", game_width, game_height, window_size_x, window_size_y);
 
 	// perform any additional initialization that requires the rendering environment to be set up
-	if (use_external_music) music_init();
+	music_init();
```

Comment:
Is there a reason why the flag is now gone?

Reply:
There is a music bug patch in music_init, this bug is not related to external music plugin. If you want to reorganize this, it is possible :)

Reply:
Looks good to me in general, but I would be happy to ensure users can still use the original music engine when they do disable the flag.

Do you think is it feasible?

Reply:
This is already the case, I wrote the test in music.cpp. My removal of the if (use_external_music) is something I forgot to do in a previous PR of mine